app/components/viral/alert_component.html.erb

```diff
@@ -1,8 +1,13 @@
-<div class="flex p-4 mb-4 border-l-4 <%= classes %> " role="alert">
-  <span class="flex-shrink-0">
-    <%= viral_icon(name: "exclamation_circle", classes: "w-5 h-5 stroke-2") %>
-  </span>
-  <div class="ml-3 text-sm font-medium">
-    <%= message %>
+<%= render Viral::BaseComponent.new(tag: "div", **system_arguments) do %>
```

Comment:
**nitpick (blocking):** Please add testing for the updates to this component.

Reply:
Please see 1a768acc29331ec492dde769ebb72f5eeb1fdcc0.